android/crt/build.gradle

```diff
@@ -188,16 +172,31 @@ afterEvaluate {
 
                 groupId = 'software.amazon.awssdk.crt'
                 artifactId = 'aws-crt-android'
+                pom {
+                    name.set("software.amazon.awssdk.crt:aws-crt-android")
+                    description.set("Java Android bindings for the AWS SDK Common Runtime")
+                    url.set("https://github.com/awslabs/aws-crt-java")
+                    licenses {
+                        license {
+                            name.set("The Apache License, Version 2.0")
+                            url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                        }
+                    }
+                    developers {
+                        developer {
+                            id.set("aws-sdk-common-runtime")
+                            name.set("AWS SDK Common Runtime Team")
+                            email.set("aws-sdk-common-runtime@amazon.com")
+                        }
+                    }
+                    scm {
+                        connection.set("scm:git:git://github.com/awslabs/aws-crt-java.git")
+                        developerConnection.set("scm:git:ssh://github.com/awslabs/aws-crt-java.git")
+                        url.set("https://github.com/awslabs/aws-crt-java")
+                    }
+                }
                 version = android.defaultConfig.versionName
             }
-
-            debug(MavenPublication) {
-                from components.debug
-
-                groupId = 'software.amazon.awssdk.crt'
-                artifactId = 'aws-crt-android'
-                version = android.defaultConfig.versionName + '-SNAPSHOT'
```

Comment:
The way we publish doesn't support snapshot yet, so, remove the debug publish which does nothing. Haven't got deep into it. The whole idea for this PR is to get it work first, we can come back later, when/if we want to completely move to gradle.

Reply:
I believe Ian said the same thing that snapshots don't work. We don't really need to publish snapshots anyway.